openshift/templates/email-verification-service/email-verification-service-deploy.yaml

```diff
@@ -217,10 +316,10 @@ parameters:
 - name: DATABASE_SERVICE_NAME
   displayName: Database Service Name
   required: true
-  value: email-verification-postgres
-- name: EMAIL_HOST
+  value: postgresql
+- name: BCGOV_EMAIL_HOST
   displayName: Email Host Name
-  required: true
+  required: false
```

Comment:
This is a required field, otherwise the service will not work.

Reply:
changed back to required